.github/workflows/package_app.yml

```diff
@@ -0,0 +1,52 @@
+name: Package Application
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+    paths:
+      - '**/package_app.yml'
+  release:
+    types: [ prereleased, released ]
+  workflow_dispatch:
```

Comment:
Do we need the `:` here since we are not providing anything after?

Reply:
I think we do need it.  I tried removing it and got a workflow error:

https://github.com/bdferris-v2/gtfs-validator/actions/runs/2387663309

More importantly, it seems pretty common to include the `:` in other projects:

https://sourcegraph.com/search?q=context:global+f:.github/workflows+workflow_dispatch&patternType=regexp

Reply:
Perfect, thank you!